CMakeLists.txt

```diff
@@ -1,28 +1,25 @@
 ########################################################################
 # Preamble
 ########################################################################
-
 cmake_minimum_required( VERSION 3.14 )
 project( ACEtk LANGUAGES CXX )
 
-
 ########################################################################
 # Project-wide setup
 ########################################################################
 
 set( CMAKE_CXX_STANDARD 17 )
 set( CMAKE_CXX_STANDARD_REQUIRED YES )
 
-option( ACEtk_unit_tests
-    "Compile the ACEtk unit tests and integrate with ctest" ON
-    )
+option(ACEtk_unit_tests "Build the ACEtk unit tests" OFF)
+option( ACEtk.python "Build ACEtk python bindings" ON )
 option( strict_compile
     "Treat all warnings as errors." ON
     )
 
 # Compile flags
-set( common_flags "-Wall" "-Wextra" "-Wpedantic" )
-set( strict_flags "-Werror" )
+set( common_flags "-Wall" "-Wextra" "-Wpedantic" ) # i'm not sure why we manage flags this way...
+set( strict_flags "-Werror" )                      # improving this will need to happen in the future for spack
```

Comment:
Nathan is going to update the build system at some point to better handle the compilation flags (and other things).

I think this still comes from the old metaconfigure system.